contributors/index.md

```diff
@@ -85,6 +85,8 @@ Helmet has help from a lot of contributors. Thanks to everyone on this list!
     - [helmet#345](https://github.com/helmetjs/helmet/pull/345)
   - [Mo Binni](https://twitter.com/mobinni)
     - [helmet#119](https://github.com/helmetjs/helmet/pull/119)
+  - [Najam Ul Saqib](https://njmulsqb.github.io)
+    - [lintJS](https://github.com/helmetjs/helmet/pull/354)
```

Comment:
Could you change this to match the others?

```suggestion
    - [helmet#354](https://github.com/helmetjs/helmet/pull/354)
```

Reply:
Done